util/fs/fs_test.go

```diff
@@ -82,48 +94,54 @@ func TestFSDetectorResize(t *testing.T) {
 
 			// Unformatted case.
 			if fsType, err := Detect(fakeDevicePath); err != ErrNoFilesystemDetected {
-				t.Fatalf("Expected ErrNoFilesystemDetected from Detect for unformatted device, but got fsType=%s err=%+v", fsType, err)
+				t.Fatalf("Expected ErrNoFilesystemDetected from Detect() for unformatted device, but got fsType=%v err=%+v", fsType, err)
```

Comment:
err is formatted with %+v here, %s  and %v in different parts of the code. If the decision is to print strings with default format and not type specifiers, then also print errors with %v since it implements the GoStringer interface as well.